compiler/damlc/daml-ide-core/test/Development/IDE/State/API/Testing.hs

```diff
@@ -9,7 +9,9 @@
 
 -- | Testing framework for Shake API.
 module Development.IDE.Core.API.Testing
-    ( ShakeTest
+    ( ShakeTest (..)
```

Comment:
I would prefer to not expose all those internals.
Instead, add a new combinator to this module something like `expectTemplateProps :: NormalizedFilePath -> [TemplateProp] -> ShakeTest ()` that takes care of all of this.

Reply:
Done